lbry/wallet/manager.py

```diff
@@ -79,11 +81,14 @@ def accounts(self):
         await asyncio.gather(*(
             l.start() for l in self.ledgers.values()
         ))
+        self.usage_payment_service = WalletServerPayer(self.ledger, self.default_wallet)
```

Comment:
Can this be instantiated in `__init__` then you won't need to add `Optional` to the type?